src/main/java/com/facebook/openwifirrm/modules/aggregators/Aggregator.java

```diff
@@ -0,0 +1,14 @@
+package com.facebook.openwifirrm.modules.aggregators;
```

Comment:
this should not be in `modules`

Reply:
Any suggestions for the best place for this?

Reply:
"com.facebook.openwifirrm.aggregators" for now